packages/cactus-test-tooling/src/main/typescript/fabric/fabric-test-ledger-v1.ts

```diff
@@ -329,6 +342,8 @@ export class FabricTestLedgerV1 implements ITestLedger {
         "10051/tcp": {}, // peer1.org2.example.com
       },
 
+      Env: [`FABRIC_VERSION=${this.fabricVersion}`],
```

Comment:
If everything is done as I suggested in my above comments then here you can just plug in the array of strings you generate from the envVars Map<string, string>

Reply:
Incorporated the changes. Please have a re-look.